Removes some NanoMed Plus products, tweaks mannitol and mutadone pills, renames healing patch

Miraviel opened the pull request:
## What Does This PR Do

Removes some NanoMed Plus products to make it less bloated and make it easier for new doctors to use it. Tweaks two pills and a patch, reasoning below.

- Renames "healing patch" to "brute patch"
- Changes "mutadone pill" from 20u to 1u
- Changes "mutadone pill" sprite
- Changes "mannitol pill" from 20u to 10u
- Changes "mannitol pill" sprite
- Replaces "health analyzer" with its advanced variant
- Removes "diphenhydramine bottle"
- Removes "ether bottle"
- Removes "insulin syringe"

## Why It's Good For The Game

**Renames "healing patch" to "brute patch"**

We have "healing" and "burn" patches, makes no sense. This way it is consistent.

**Changes "mutadone pill" from 20u to 1u and its sprite**

One unit is enough of this medicine. Changed its sprite from the generic white pill to the blue-purple sprite, as genetics itself is blue-purple as well.

**Changes "mannitol pill" from 20u to 10u and its sprite**

20 units is an overkill, we really do not need megapills. Sprite is changed because white is boring and non-descriptive, and the new yellow is not associated with any damage type either.

**Replaces "health analyzer" with its advanced variant**

The analyzer and its upgrade are almost always vended together anyhow, this way we have one less entry on the list (by removing the upgrade) and saves new doctors from accidentally using the less descriptive one.

**Removes "diphenhydramine bottle"**

Has four uses, 1) purging histamine, without an easy access to it, histamine might be actually dangerous again 2) virology removes symptoms with it, they already start with a full bottle 3) used to make crank and krokodil, not quite something NT would encourage their medical chemists to do, 4) it is an advanced virus cure, this way chemistry might get involved in the round past the 00:10 minutes mark by Medbay not having an easy access to it.

**Removes "ether bottle"**

Ether is godawful as a surgical sedative, nobody came up with a good usage for it, it can be re-added when someone makes it viable again. Its antag potential is not great either, you can get pancuronium and sulfonal from hacked nanomeds anyhow and sulfonal works twice as fast as ether.

**Removes "insulin syringe"**

Purges sugar and helps creating aranesp. It's one more thing that bloats the already huge product list the NanoMed has and if you want drugs as a medical chemist, work for it.

## Images of changes

![image](https://user-images.githubusercontent.com/33333517/187031649-ca3a1f1f-3a19-422e-aadb-5afef5645e10.png) 
![image](https://user-images.githubusercontent.com/33333517/187031655-00b4e8b8-b054-4f40-801f-1ff920a28065.png)

## Testing
Launched the Cyberiad, walked up to a NanoMed Plus, checked its contents.

## Changelog
:cl:
tweak: Renamed "healing patch" to "brute patch".
tweak: Mannitol pill and mutadone pill sprites are changed from the boring white ones.
tweak: Mannitol pills changed from 20u to 10u, mutadone pills from 10u to 1u.
tweak: Replaced the health analyzer and the health analyzer upgrade with the upgraded health analyzer variant.
del: Removed the following from NanoMed plus: diphenhydramine bottle, ether bottle, insulin syringe.
/:cl:

meow20 commented:
Absolutely goated PR, im surprised i didn't went and did it myself

CinnamonSnowball commented:
Truly based PR; making Mannitol not a total brain death cure in a single pill might make it relevant for Chemistry to make extra, same with Mutadone not being just a mindless "grab one pill from the vendor, repill into twenty times the amount"

Personally, I feel the yellow pill could be mistaken to be associated with the yellow-themed burn damage type instead of brain damage, though? IMO the gray-red palette fits brain damage fine; since it doesn't have a designated color, white/gray is kind of the default, and one could even stretch it to be reminiscent of a grayish-red brain with a darker red inside all the folds

Contrabang commented:
Clone damage is usually associated with purple in the few instances that it is colored iirc, so I think mutadone's color is probably good but I agree that mannitol should probably stay grey because grey matter lol.

Also you might want to change the icon for the health analyzer now that it includes the upgrade in it? Also there should probably still be someway to get an analyzer upgrade without having to go all the way to science for one, because if you have to get a health scanner from a medkit, theres no extra upgrades in medbay now. Might be a dumb idea but maybe make the health analyzer upgrades be able to be bought with coins from the Emergency Nanomachines? They only have 4 items so adding a 5th coin one wouldn't be so bad I think.

Miraviel commented:
Update:

- @hal9000PR requested not to remove insulin as aranesp is actually used, moved it to contraband (not the best solution, but at least it is out of the product list, I'm open for ideas for this one) [2e2b2ba](https://github.com/ParadiseSS13/Paradise/pull/18904/commits/2e2b2ba880a0f70ef71fe8f72c1aea74dec60c8a)

@CinnamonSnowball 

> IMO the gray-red palette fits brain damage fine; since it doesn't have a designated color, white/gray is kind of the default, and one could even stretch it to be reminiscent of a grayish-red brain with a darker red inside all the folds. Also, the Mannitol chemical itself is white, so I guess that checks out?

Reverted the mannitol pill sprite change https://github.com/ParadiseSS13/Paradise/pull/18904/commits/5bc98dbb290fed1ea58d6b09e88f0e29c8c9ae9d

> Edit: side note, would renaming some of the things offered in the vendor to standardize the capitalization fit in the scope of the PR? We have Charcoal Bottles but also salicylic acid bottles and even Salbutamol pills

Done for syringes too, https://github.com/ParadiseSS13/Paradise/pull/18904/commits/0833bcc400986081e6031cef7baa829f0ed936c6 https://github.com/ParadiseSS13/Paradise/pull/18904/commits/ffd57b9e9484bb2632d1749ad32ad099e0d09420 I was not sure about the pills because Salbutamol is ... kind of the name of the medicine? I think??? If someone can correct me on this, I'll rename the pills too 

@Contrabang 

> Also you might want to change the icon for the health analyzer now that it includes the upgrade in it?

I renamed it from "health analyzer" to "advanced health analyzer", noticed that was off too. https://github.com/ParadiseSS13/Paradise/pull/18904/commits/abdfd7081f6da4ba2e9db180e328ed36a38d135a As for the icon, sadly I cannot do it, upgraded health analyzers have the icon of regular health analyzers, and get an overlay upon init. The product list works with the original icons. If anyone knows how to fix it, let me know.

> Also there should probably still be someway to get an analyzer upgrade without having to go all the way to science for one, because if you have to get a health scanner from a medkit, theres no extra upgrades in medbay now. Might be a dumb idea but maybe make the health analyzer upgrades be able to be bought with coins from the Emergency Nanomachines? They only have 4 items so adding a 5th coin one wouldn't be so bad I think.

I was considering this, but wall nanomeds are everywhere, we would allow a lot of people to get upgraded analyzers this way. :gatto:

CinnamonSnowball commented:
> Salbutamol is ... kind of the name of the medicine?
That's actually a very good point; yeah, they can stay as is then

Sirryan2002 reviewed (approved):
code looks fine
